sync/tx_pool_sync.md

```diff
@@ -0,0 +1,58 @@
+[back](../SYNC.md)
+
+# TX-pool synchronization
+
+On startup the epoch node needs to update (or get from scratch if it is a
+completely new node) its list of unconfirmed transactions. New transactions it
```

Comment:
Minor nitpick: people are generally more familiar with the term »mempool« at this point, so I would suggest using that here.
Unconfirmed transactions could possibly also refer to transactions, which were included in the latest block.